modin/pandas/series.py

```diff
@@ -486,6 +486,9 @@ def values(self):  # noqa: RT01, D200
         """
         import modin.pandas as pd
 
+        if isinstance(self.dtype, pandas.core.dtypes.dtypes.ExtensionDtype):
```

Comment:
Modin doesn't have core `module`.

Reply:
this should make the check on L493 unnecessary

Reply:
@jbrockmendel thanks! It was a mistake. Fixed.